data-prepper-plugins/rds-source/src/test/java/org/opensearch/dataprepper/plugins/source/rds/leader/LeaderSchedulerTest.java

```diff
@@ -138,6 +140,6 @@ void test_shutDown() {
     }
 
     private LeaderScheduler createObjectUnderTest() {
-        return new LeaderScheduler(sourceCoordinator, sourceConfig, schemaManager, dbMetadata);
+        return new LeaderScheduler(sourceCoordinator, sourceConfig, s3Prefix, schemaManager, dbMetadata);
```

Comment:
Are we storing S3 partition info in Source coordinator ? If yes, we can try to assert S3 prefix value in S3 Partition.

Reply:
S3 partition info is not stored in source coordinator.

Reply:
Added a unit test to assert the s3 prefix